admin/class-boldgrid-backup-admin-settings.php

```diff
@@ -691,6 +691,14 @@ private function update_settings() {
 				}
 			}
 
+			/*
+			 * Save "backup filelist analysis" setting.
+			 *
+			 * @since SINCEVERSION
+			 */
+			$tmp_key              = Boldgrid_Backup_Admin_Filelist_Analyzer::$settings_key;
```

Comment:
why is this named $tmp_key?
It makes me feel like this key is only temporary, and added a lot of confusion when looking at the code.
I get that you are obtaining this value from the Boldgrid_Backup_Admin_Filelist_Analyzer::$settings_key, but it seems like a lot of extra work than to just use a string literal 'backup_filelist_analysis' . I feel like it would be more clear in the code, especially in places like L700 , if you were just using the string literal instead of this.

Reply:
Sure, "filelist_analysis" is better than "tmp_key".

Reply:
@jamesros161 a variable is used so we don't repeat ourselves - https://en.wikipedia.org/wiki/Don%27t_repeat_yourself

How about this? https://github.com/BoldGrid/boldgrid-backup/commit/9642a7736fcae204663d62b16c2d7f217e4fd035

Reply:
@cssjoe I see you commented too. I think https://github.com/BoldGrid/boldgrid-backup/commit/9642a7736fcae204663d62b16c2d7f217e4fd035 is a little cleaner - what do you think

Reply:
fixed in https://github.com/BoldGrid/boldgrid-backup/commit/976a13e64e1ca3ae2cc50a64971615fac67333ac